onnx/backend/test/case/node/bitwiseand.py

```diff
@@ -38,15 +38,15 @@ def export_bitwiseand_broadcast() -> None:
         )
 
         # 3d vs 1d
-        x = np.random.randn(3, 4, 5).astype(np.uint64)
-        y = np.random.randn(5).astype(np.uint64)
+        x = np.random.randint(-10, 10, size=(3, 4, 5)).astype(np.uint64)
```

Comment:
Just wondering: is the cast from negative into to uint precisely defined? Why not generate something in the valid range in the first place? We could do it in one place by defining a utility/helper method that takes shape and dtype as parameters, because a range like -10 to 10 may not cover all cases (depending on the op.)

Reply:
Sounds good. I have introduced create_random_int and updated by some of your reviews. PTAL. Thank you!